onnx/gen_proto.py

```diff
@@ -42,16 +45,28 @@ def process_ifs(lines, onnx_ml):
                 yield line
 
 
+IMPORT_REGEX = re.compile(r'(\s*)import\s*"([^"]*)\.proto";\s*$')
 PACKAGE_NAME_REGEX = re.compile(r'\{PACKAGE_NAME\}')
+ML_REGEX = re.compile(r'(.*)\-ml')
 
 def process_package_name(lines, package_name):
+    need_rename = (package_name != DEFAULT_PACKAGE_NAME)
     for line in lines:
-        yield PACKAGE_NAME_REGEX.sub(package_name, line)
+        m = IMPORT_REGEX.match(line) if need_rename else None
+        if m:
+            include_name = m.group(2)
+            ml = ML_REGEX.match(include_name)
+            if ml:
+                include_name = "{}_{}-ml".format(ml.group(1), package_name)
```

Comment:
Use onnx-ml_PACKAGENAME to avoid have different processing logic here?

Reply:
I'm a bit reluctant to do this because simplifying the logic here will require changes to use `onnx-ml_PACKAGENAME` at many other places.